samples/bluetooth/hci_uart/boards/nrf52833dk_nrf52833.overlay

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Wouldn't it be better to create e.g. nrf52833df_nrf52833_dtm.overlay and use it through `DTC_OVERLAY_FILE` CMAKE?
I'm not sure if DF overlay should be set to every build of `hci_uart` for nRF52833 target.

Also there should be similar configuration for nRF5340 board.

Reply:
I don't have a strong opinion about that. It seems like we can keep radio antenna configuration safely in default board overlay since it doesn't have any impact on the Kconfig or source files, when building with DF feature disabled but maybe I missed something. I added also antenna configuration for `nrf5340dk_nrf5340_cpunet` target. I can prepare separated overlays files for this if you want.

Reply:
In my humble opinion `hci_uart` should be a general sample that is oriented for common functionality, hence the DF and test commands should be enabled optionally (explicitly by end user).

Reply:
Ok I added a special target like nrf5340dk_nrf5340_cpunet_df. You can build this sample with df support by invoking for example: `west build -b nrf5340dk_nrf5340_cpunet@df`